config/initializers/mobility.rb

```diff
@@ -0,0 +1,6 @@
+Mobility.configure do |config|
```

Comment:
Do we need to initialize this at all, given it's done in Spina? I'd suggest we should't be duplicating this into engines.